app/views/content/complaints.md

```diff
@@ -0,0 +1,13 @@
+The Department for Education is committed to providing a high level of customer service, and takes complaints seriously. Letting us know if something is wrong means we can fix it and improve the service for everyone.
+
+##Giving feedback
+
+If you have feedback about the service, or want to give suggestions for improvement, you can email us as [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk).
```

Comment:
```suggestion
If you have feedback about the service, or want to give suggestions for improvement, you can email us as <becomingateacher@digital.education.gov.uk>.
```

Reply:
Neat, I didn't know you could do that. Cheers @paulrobertlloyd